pkg/doc/verifiable/credential_test.go

```diff
@@ -601,6 +601,7 @@ func TestCustomCredentialJsonSchemaValidator2018(t *testing.T) {
 
 	missingReqFieldSchema, mErr := json.Marshal(raw)
 	require.NoError(t, mErr)
+	fmt.Println(string(missingReqFieldSchema))
```

Comment:
why Println instead of t.Logf?

https://golang.org/pkg/testing/#B.Logf

Reply:
This is my negligence, I will fix it.